### DISCUSSION TITLE
Platform specific extensions

### DISCUSSION THREAD

**Discussion by isidorn** [microsoft/vscode-discussions#23] [Announcements]
Extensions can publish different VSIXs for each platform (Windows, Linux, macOS) VS Code is running on. We call such extensions platform-specific extensions. More details can be found in our docs:
https://code.visualstudio.com/api/working-with-extensions/publishing-extension#platformspecific-extensions

Let us know if you have any questions around platform specific extensions.

fyi @saikanth007

**Comment by ericdrobinson** [comment]
I [asked a question](https://github.com/microsoft/vscode-discussions/discussions/19) about how Platform Specific Extensions are handled by VS Code (outside of the Marketplace publishing mechanism). [The answer](https://github.com/microsoft/vscode-discussions/discussions/19#discussioncomment-2891009) is very revealing and may be helpful for extension developers to review.
